documentation/course/E1.md

```diff
@@ -586,25 +585,129 @@ Reguły zostały podzielone według encji, których dotyczą oraz są uporządko
 
 ## Prototypy interfejsu
 
-<!-- [For crucial functional requirements in each domain (at least 2). It is recommended, the prototypes be a part of user-story specification] -->
+### Wymaganie 1 (Tomasz Chojnacki)
 
-### Wymaganie 1
+1. `ACC/01` - Jako _gość_ chcę mieć możliwość rejestracji _konta_ _pasażera_. 
 
-TODO@tchojnacki
+#### Mapa nawigacyjna
+
+```mermaid
+classDiagram
+    direction LR
+    class EkranRejestracji {
+        <<ServerPage>>
+        imieINazwisko
+        email
+        haslo
+        powtorzHaslo
+    }
+    class LoginScreen["Ekran logowania"] { <<ServerPage>> }
+    class MainScreen["Ekran główny"] { <<ServerPage>> }
+
+    EkranRejestracji ..> LoginScreen: «Navigates»\n«EndUseCase»\n{ event = btnPrzejdzDoLogowania }
+    EkranRejestracji ..> MainScreen: «Navigates»\n«EndUseCase»\n{ event = btnZarejestrujSie }
+    LoginScreen ..> EkranRejestracji: «Navigates»\n{ event = btnPrzejdzDoRejestracji }
+    LoginScreen ..> MainScreen: «Navigates»\n{ event = btnZalogujSie }
+```
+
+#### Prototypy ekranów
+
+![](images/Account1.png)
```

Comment:
**nit**: images should have alt-names

I'll point it out only for the first occurrence, but my point stands for all of them.